specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Line 320
```
  rla_tabulation_outcome: TYPE FROM election_outcome
```
I'm not sure what this is for.  Is it a corrected `reported_tabulation_outcome` based on audit discrepancies?